src/components/header/_header.scss

```diff
@@ -12,7 +12,8 @@
     height: auto;
     fill: $color-white;
 
-    @include mq(xs, s) {
+    @include mq(xs, 399px) {
+      padding: 0.45rem 0;
       max-width: 145px;
```

Comment:
Could we increase this slightly, to 160px?  Do we need the padding?

Reply:
The padding is to make up the difference between the svg logo used in the census style and the text used in the regular style. Another option would be to try look at font sizes compared to the svg size and try to match the size of them including any whitespace in the svgs